protocols/xml/src/main/java/org/opennms/protocols/json/collector/AbstractJsonCollectionHandler.java

```diff
@@ -61,6 +58,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import net.sf.json.JSON;
+import net.sf.json.JSONObject;
+import net.sf.json.JSONSerializer;
```

Comment:
someday, we'll keep all our various editors from doing the import dance, but today is not that day <insert aragon meme here>